ups/eupspkg.cfg.sh

```diff
@@ -41,16 +41,17 @@ install()
     fi
 
     (
-        # Install packages on which the stack is know to depend
-        # Note: it's not clear if agreement was reached to use all of these,
-        #       or they crept in by accident.
-
-        export PATH="$PREFIX/bin:$PATH"
-        # XXX the list of conda packages should be kept in sync with
-        # https://github.com/lsst/lsstsw/blob/master/bin/deploy
-        # The lsstsw list of packages should be considered authoritative
-        conda install --yes numpy scipy matplotlib requests cython sqlalchemy astropy pandas
-        pip install stsci.distutils
+        # Install packages on which the stack is known to depend
+
+        export PATH="$LSSTSW/miniconda/bin:$PATH"
+        local tmpdir=${TMPDIR:-/tmp}
+        local reqfile='conda_packages.txt'
+        # XXX change me before merge to master
+        # local baseurl='https://raw.githubusercontent.com/lsst/lsstsw/master/etc/'
+        local baseurl='https://raw.githubusercontent.com/jhoblitt/lsstsw/tickets/DM-5105-pin-python-package-versions/etc'
```

Comment:
The "correct" URL won't work until https://github.com/lsst/lsstsw/pull/82 is merged.  I left the PR this way so it could be tested without modification.  I will update it shortly.